fhir-server/src/test/java/org/linuxforhealth/fhir/server/util/FHIRRestHelperTest.java

```diff
@@ -2984,8 +2985,8 @@ public void testTransactionBundlePostWithResourceSchemeReference() throws Except
      */
     @Test
     public void testSearchWithPreviousPageResultsShiftWarning() throws Exception {
+        Random random = new Random();   
```

Comment:
Use of Random is going to be flagged by code-scanner. Should always use SecureRandom.

Reply:
This has been fixed as suggested.